docs/release-process.md

```diff
@@ -0,0 +1,42 @@
+# Application Services Release Process
+
+These are the steps needed to cut a new release.
+
+1. Update the changelog.
+
+    1. Add a new header between the "Full Changelog" and "See the release process docs..." lines with the version number and date of the next release.
+    2. Replace `master` in the Full Changelog link to be `v<new-version-number>`. E.g. if you are releasing 0.13.2, the link should be
+
+        ```
+        [Full Changelog](https://github.com/mozilla/application-services/compare/v0.13.1...v0.13.2)
+        ```
+
+        Note that this needs three dots (`...`) between the two tags (two dots is different). Yes, the second tag doesn't exist yet, you'll make it later.
+
+    3. Add a new "Full Changelog" link that starts at your new version and continues to master. E.g. for 0.13.2 this would be
+        ```
+        [Full Changelog](https://github.com/mozilla/application-services/compare/v0.13.2...master)
+        ```
+        Again, this needs 3 dots.
+    4. Optionally, go over the commits between the past release and this one and see if anything is worth including.
```

Comment:
Is this kind of a "sanity-check that we didn't miss any important changes" step, but we expect that the changelog is already up-to-date?

Reply:
Yes.